.babelrc

```diff
@@ -2,5 +2,6 @@
   "presets": ["env"],
   "plugins": [
     ["transform-replace-object-assign", { "moduleSpecifier": "object.assign" }],
+    "transform-object-rest-spread",
```

Comment:
is this not included in env?

Reply:
I don't think so (I got an error when using spread syntax). It certainly wasn't there when I was on babel 6, not sure if the babel 6 `env` is updated.